bin/eds-apply-theme.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+(async function () {
+  const StyleDictionary = require('style-dictionary');
+  const path = require('path');
+  const fs = require('fs');
+  const { getConfig } = require('./_util');
+
+  let packageRootPath;
+  try {
+    packageRootPath =
+      path.dirname(require.resolve('@chanzuckerberg/eds')) + '/tokens/json/';
+  } catch (e) {
+    console.error('EDS package not installed. Using local path...');
+    packageRootPath =
+      path.dirname(require.main.path) + '/src/tokens-dist/json/';
```

Comment:
what do we expect to be at this directory if eds is not installed?

Reply:
This little bit is essentially for testing WITHIN EDS. I tested most of the functionality by dog-fooding this in the repo. I can update the error message to be `info` or something, and make this say something closer to that